awx/main/tasks.py

```diff
@@ -224,11 +315,15 @@ def cluster_node_heartbeat(self):
             instance_list.remove(inst)
     if this_inst:
         startup_event = this_inst.is_lost(ref_time=nowtime)
-        if this_inst.capacity == 0:
+        if this_inst.capacity == 0 and this_inst.enabled:
```

Comment:
Would be nice to create a property like `is_available` to put this kind of `.capacity` `.enabled` logic pairing.